js/README.md

```diff
@@ -100,55 +100,163 @@ Subdirectories with additional utilities
 This section describes how to add new blocks to Music Blocks in order
 to add functionality.
 
-* You can add an individual block to a palette by modifying two files:
-`basicblocks.js` and `logo.js`.
+<!-- * You can add an individual block to a palette by modifying two files:
+`basicblocks.js` and `logo.js`. -->
+* To add a new block you first need to look through `js/blocks/` and determine if a new file is needed or not. If required then go through the section down below on how to declare new files for blocks requiring so.
```

Comment:
A new file is needed if you want to add a new palette. Otherwise, modify the file associated with the palette to which you want to add the new block.

Reply:
I've updated the explanation. Please have a look.

Reply:
It is still not clear that "new file" corresponds to "new palette"

Reply:
I've changed it to this now.
Will it work?

![Screenshot from 2020-02-02 21-00-18](https://user-images.githubusercontent.com/44023445/73610587-146ebc80-45ff-11ea-9f76-664777a0222c.png)